README.md

```diff
@@ -67,7 +67,7 @@ Use '-p' flag to configure prefix, '-d' flag to select db file, and '-f' flag to
 
 For example,
 <pre>
-    $ ./build/ndnfs-server -p /ndn/broadcast/ndnfs -l ndnfs-server.log -f /tmp/ndnfs -d ndnfs.db
+    $ ./build/ndnfs-server -p /ndn/broadcast/ndnfs -l ndnfs-server.log -f /tmp/ndnfs -d /tmp/ndnfs.db
```

Comment:
I suggest not making this change

Reply:
I rewrited this because I think this instruction is confusing. By default, ndnfs uses /tmp/ndnfs.db. However the example here uses ./ndnfs.db. So if people run the the examples as indicated:
$ mkdir /tmp/ndnfs
$ mkdir /tmp/dir
$ ./build/ndnfs -s -f /tmp/dir /tmp/ndnfs
$ ./build/ndnfs-server -p /ndn/broadcast/ndnfs -l ndnfs-server.log -f /tmp/ndnfs -d ndnfs.db
$ echo "Hello, world!" > /tmp/ndnfs/test.txt
$ ndnpeek -pf /ndn/broadcast/ndnfs/test.txt
There will be an error saying no such file in /tmp/ndnfs.